docs/source/notebooks/release_announcement.ipynb

```diff
@@ -0,0 +1,2774 @@
+{
```

Comment:
<ul><li>I think we should add the link tot he paper https://research.google/pubs/pub46001/</li><li>Shall we comment on why it makes sense to have mmm and clv in the same package? Something along the lines: to fully understand the return on investment of marketing activities one should not only consider ROAS from an MMM but also couple it with CLV estimations to have a more complete view on the return over a long period of time.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Sounds great, adding it</p>